p2p/src/net/mock/transport/impls/stream_adapter/identity.rs

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) 2022 RBB S.r.l
+// opensource@mintlayer.org
+// SPDX-License-Identifier: MIT
+// Licensed under the MIT License;
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// https://github.com/mintlayer/mintlayer-core/blob/master/LICENSE
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use futures::future::{ready, BoxFuture};
+
+use crate::net::mock::{peer::Role, transport::PeerStream};
+
+use super::StreamAdapter;
+
+#[derive(Debug)]
+pub struct IdentityStreamAdapter;
```

Comment:
I will take a note, thanks.
But `new` here is only because of the trait requirement (and this part will probably change soon).
Let's please keep it for now.